lib/inherited_class_var.rb

```diff
@@ -60,7 +62,7 @@ def hidden_variable_name(variable_name)
     # @return [Array<Module>] inherited_ancestors of included_module (including self)
     def inherited_ancestors(included_module=InheritedClassVar)
       included_model_index = ancestors.index(included_module)
-      included_model_index == 0 ? [included_module] : ancestors[0..(included_model_index - 1)]
+      included_model_index == 0 ? [included_module] : (ancestors[0..(included_model_index - 1)] - [InvalidOptions])
```

Comment:
ah... i see. forgot about that concern. i think we can keep it anyway :P (otherwise we have a whack-a-mole situation whenever we include anything)